Makefile

```diff
@@ -64,3 +64,11 @@ test_quality: ## Run Quality checks
 	pylint submissions
 	isort --check-only --recursive submissions manage.py setup.py settings.py
 	pycodestyle . --config=pycodestyle
+
+##################
+#Devstack commands
+##################
+
+install-local-submissions: ## installs your local submissions code into the LMS and Studio python virtualenvs
```

Comment:
I like that, HOWEVER, (excuse the caps :) ) submissions must be installed into ora2's virtual environment inside lms in order for it to pick up the latest/local version.